### PR TITLE
Add org.torproject.torbrowser-launcher

### PR DESCRIPTION
**Pull request by AsciiWolf**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I have [built][build] and tested the submission locally.
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. (**I am an upstream contributor.**)
- [x] The domain used for the application ID is controlled by the application developers either directly or through the code hosting (e.g. GitHub, GitLab, SourceForge, etc.). The [application id guidelines][app-id] are followed.
- [x] Any additional patches or files have been submitted to the upstream projects concerned. (**The app prefix patch is downstream specific.**)

This is just a rename of the [com.github.micahflee.torbrowser-launcher](https://github.com/flathub/com.github.micahflee.torbrowser-launcher) Flatpak. I will EOL the com.github.micahflee.torbrowser-launcher Flatpak after this PR gets merged.

More information: https://github.com/flathub/com.github.micahflee.torbrowser-launcher/issues/176

/cc @boklm @micahflee
(Please, add push rights to these two.)

[reqs]: https://docs.flathub.org/docs/for-app-authors/requirements
[maint]: https://docs.flathub.org/docs/for-app-authors/maintanance
[submission]: https://docs.flathub.org/docs/for-app-authors/submission
[build]: https://docs.flathub.org/docs/for-app-authors/submission/#before-submission
[app-id]: https://docs.flathub.org/docs/for-app-authors/requirements#application-id

### CODE REVIEW

**Comment by AsciiWolf** [comment]
bot, build

**Comment by AsciiWolf** [comment]
> Build [93761 successful](https://buildbot.flathub.org/#/builders/6/builds/93761)

Tested and everything seems to work fine. :-)

File: org.torproject.torbrowser-launcher.yaml [hunk 1]
```diff
@@ -0,0 +1,217 @@
...
+separate-locales: false
+command: torbrowser-launcher
+rename-desktop-file: torbrowser.desktop
+rename-icon: torbrowser
+finish-args:
+  - --share=ipc
+  - --share=network
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --talk-name=org.freedesktop.Notifications
```
Comment (bbhtt) [inline org.torproject.torbrowser-launcher.yaml:21]:
```suggestion
```

Not needed with libnotify >= 0.8.0 it uses the portal by default

File: org.torproject.torbrowser-launcher.yaml [hunk 2]
```diff
@@ -0,0 +1,217 @@
...
+  - --device=dri
+  - --socket=pulseaudio
+  - --filesystem=xdg-download
+  - --env=MOZ_ENABLE_WAYLAND=1
```
Comment (bbhtt) [inline org.torproject.torbrowser-launcher.yaml:25]: Why not use the ESR default?

File: org.torproject.torbrowser-launcher.yaml [hunk 3]
```diff
@@ -0,0 +1,217 @@
...
+        x-checker-data:
+          type: anitya
+          project-id: 6359
+          url-template: https://gitlab.torproject.org/tpo/applications/torbrowser-launcher/-/archive/v$version/torbrowser-launcher-v$version.tar.gz
+      - type: patch
+        path: use-flatpak-sys-prefix.patch
+    build-commands:
+      - python3 setup.py install --prefix=${FLATPAK_DEST} --root=/ --optimize=1
+    post-install:
+      - sed -i 's:Icon=torbrowser:Icon=org.torproject.torbrowser-launcher:' /app/share/applications/torbrowser-settings.desktop
```
Comment (bbhtt) [inline org.torproject.torbrowser-launcher.yaml:216]: Use `desktop-file-edit` here please

File: org.torproject.torbrowser-launcher.yaml (hunk 2 above)
Comment (AsciiWolf) [reply org.torproject.torbrowser-launcher.yaml:25]: As far as I know, Tor Browser still defaults to not enabling Wayland by default. This is however not what we want in the Flatpak'ed version.

Comment (bbhtt) [reply org.torproject.torbrowser-launcher.yaml:24]: Would be better to follow the upstream default, imo, but up to you

**Comment by AsciiWolf** [comment]
bot, build org.torproject.torbrowser-launcher

**Pull request closed without merging**